### DISCUSSION TITLE
Feature Request: Required Field Indicators

### DISCUSSION THREAD

**Discussion by scottanthonymurray** [indiana-university/rivet-source#414] [Feature and enhancement requests]
Team - From what I can tell, there isn't a standard to indicate required input fields (vs optional).  It would be helpful if Rivet set that standard so we become more uniform in an approach throughout IU.  

Thanks!!!!

**Comment by levimcg** [comment]
Ran across this article last week. I haven't had a lot of time to look over it, but seems like there might be some good stuff in here to reference when we get to this issue.

https://developer.paciellogroup.com/blog/2019/02/required-attribute-requirements/
